wifinina/tcp.go

```diff
@@ -24,6 +49,9 @@ type Driver struct {
 	proto uint8
 	ip    uint32
 	port  uint16
+
+	accessPoints []accessPoint
```

Comment:
It is a good idea to store the information of the access point. This makes it easy to reconnect, unlike rtl8720dn and older wifinina.
I don't know if it should be stored in wifinina.Driver or in wifinina.Device.

I would like to know if there is a good reason.

Reply:
It's a bit of definitions mix here.
So, my reasoning goes like this: 
- `wifinina.Device` is about talking to the chip, lower level. It is a "driver" for this chip, in spirit of this repo.
- `wifinina.Driver` is about networking, more like adapter actually, making `wifinina.Device` usable in TinyGo, with `net`, `net/tls` and `net/http` packages.

In this light, access point information belongs to higher level, means `wifinina.Driver`.

Reply:
I understood.
I think it is good as it is now.